### ISSUE TITLE
data_matching none is not working

### ISSUE THREAD

**Issue by killenb** [ChimeraTK/ControlSystemAdapter-DoocsAdapter#39]
Observed in llrfCtrl7 in single cavity configuration.

Variables in the "publish all" location "UNMAPPED" permanently give the warning "data loss due to failed matching", although `data_matching` has been set to `none` for the whole server and explicitly also for this location:
```
WARNING: Data loss in scalar property UNMAPPED.POLARIX/Controller.PulseShape.Flattop1.startTime due to failed data matching between value and macro pulse number.
WARNING: Data loss in scalar property UNMAPPED.POLARIX/Controller.PulseShape.Flattop1.length due to failed data matching between value and macro pulse number.
```
In fact, it would be wrong to publish because only the macro pulse number has changed, and the data is not strongly correlated to it.

In this sense the current implementation is working correctly, but is spamming the log.

**Comment by ckampm** [comment]
Started evaluating this the wip_ckampm_issue39_data_matching branch.

**Comment by ckampm** [comment]
Solved in bb3c2cc
